int32 quality
int32 lane_type
float64[] c
float64 marking_width
float64 view_range
bool view_range_availability